daemon/logger/jsonfilelog/jsonlog/jsonlog.go

```diff
@@ -14,6 +14,8 @@ type JSONLog struct {
 	Created time.Time `json:"time"`
 	// Attrs is the list of extra attributes provided by the user
 	Attrs map[string]string `json:"attrs,omitempty"`
+	// Tags requested the operator
+	Tag string `json:"tag,omitempty"`
```

Comment:
I don't think we need this field now?

Reply:
@cpuguy83 Ah yes. Let me update the PR shortly. Thanks!